release.js

```diff
@@ -36,15 +36,30 @@ module.exports = async function ({ github, context, inputs }) {
 
   const { opticUrl, npmTag, version, id } = releaseMeta
 
+  const run = runSpawn()
   if (!pr.merged) {
-    return github.rest.repos.deleteRelease({
-      owner,
-      repo,
-      release_id: id,
-    })
+    const branchName = `release/${version}`
+    try {
+      await run('git', ['push', 'origin', '--delete', branchName])
+    } catch (err) {
+      return core.setFailed(
```

Comment:
Could we try to remove the release even if the branch delete fail?

Reply:
That was the original plan, I don't think it's a bad idea. What do you think @simoneb ?

Reply:
yes